privaterelay/apps.py

```diff
@@ -16,5 +16,8 @@ def ready(self):
             '%s/jwks' %
             settings.SOCIALACCOUNT_PROVIDERS['fxa']['OAUTH_ENDPOINT']
         )
-        resp_json = resp.json()
-        self.fxa_verifying_keys = resp_json['keys']
+        if resp.status_code == 200:
+            resp_json = resp.json()
+            self.fxa_verifying_keys = resp_json['keys']
+        else:
+            pass  # what's the right thing to do here?
```

Comment:
```suggestion
        if resp.status_code == 200:
            resp_json = resp.json()
            self.fxa_verifying_keys = resp_json['keys']
```
If we let this code finish and leave `fxa_verifying_keys` as `[]`, then we'll throw `Could not authenticate JWT with FXA key.` exceptions later when we try to use the `fxa_verifying_keys` value that doesn't exist. That's probably the right behavior ...

1. The exceptions will be logged into Sentry, 
2. FXA will retry the callbacks until the callback is successful, 
3. The code in `_authenticate_fxa_jwt` already tries to call this `ready` code again, so:
4. When the FXA verifying keys are available again (because FXA fixes the server), this code will populate `fxa_verifying_keys`,
5. The callback will process successfully,
6. FXA will stop retrying, and finally:
7. We'll have a nice fat spike of FXA callback errors in Sentry so we know about it.


But, the best way to tighten this up is probably to add a check in `_verify_jwt_with_fxa_key` ... ?

```python
  def _verify_jwt_with_fxa_key(req_jwt, private_relay_config):
      if private_relay_config.fxa_verifying_keys == None:
        raise Exception("FXA verifying keys are unavailable.")
      ...
```